src/components/CharacterBox/CharacterBox.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react'
 import CharacterCard from '../CharacterCard/CharacterCard'
+import fakeCharacters from '../../mockData/fakeCharacters.js'
```

Comment:
We don't use this dataset in this component, maybe it is better to delete it?

Reply:
Good catch, just removed it